install/installer/pkg/components/server/types.go

```diff
@@ -32,7 +32,7 @@ type ConfigSerialized struct {
 	ImageBuilderAddr                  string   `json:"imageBuilderAddr"`
 	VSXRegistryUrl                    string   `json:"vsxRegistryUrl"`
 	ChargebeeProviderOptionsFile      string   `json:"chargebeeProviderOptionsFile"`
-	StripeSettingsFile                string   `json:"stripeSettingsFile"`
+	StripeSecretsFile                 string   `json:"stripeSecretsFile"`
```

Comment:
nit: Why is this called "StripeSecretsFile"? Given the rename, shouldn't it be the "StripeApiKeyFile" or so? :thinking:

Reply:
yes, perhaps.  I went with the more general name because I wasn't sure if we'd need other secrets in there besides the keys.

Reply:
This is about the file, so "secret mount point" + "key inside the secret".
If we added another key in that secret it would still be another file.

Reply:
Anyway, I just noticed it already got merged. But still think `StripeKeysFile` would make sense. :upside_down_face: